hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -143,6 +143,9 @@
   public static final String CLIENT_HEARTBEAT_NUM_TOLERABLE_MISSES_PROP = "hoodie.client.heartbeat.tolerable.misses";
   public static final Integer DEFAULT_CLIENT_HEARTBEAT_NUM_TOLERABLE_MISSES = 2;
 
+  public static final String COLLECT_OBSERVABILITY_METRICS = "hoodie.collect.observability.metrics";
+  public static final String DEFAULT_COLLECT_OBSERVABILITY_METRICS = "true";
```

Comment:
default false is better unless we are use this does not cause scalability issues with the reporter/metrics platform.

Reply:
Done